DependencyInjection/JMSDiExtraExtension.php

```diff
@@ -54,6 +55,19 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('jms_di_extra.disable_grep', $config['disable_grep']);
         $container->setParameter('jms_di_extra.doctrine_integration', $config['doctrine_integration']);
 
+        if ($config['annotation_patterns']) {
+            if ($container->getParameterBag()->has('jms_di_extra.annotation_patterns')) {
```

Comment:
I don't think that we should manage this case. People should use the bundle configuration.